app/app.go

```diff
@@ -282,11 +279,13 @@ func NewDesmosApp(
 	app.App = appBuilder.Build(logger, db, traceStore, baseAppOptions...)
 
 	// set up non depinject support modules store keys
-	app.keys = sdk.NewKVStoreKeys(
+	storeKeys := sdk.NewKVStoreKeys(
 		ibcexported.StoreKey, ibctransfertypes.StoreKey, ibcfeetypes.StoreKey,
 		icahosttypes.StoreKey, icacontrollertypes.StoreKey, wasmtypes.StoreKey,
 	)
-	app.MountKVStores(app.keys)
+	for _, key := range storeKeys {
+		app.RegisterStores(key)
+	}
```

Comment:
In the new version, stores can be set by new function `RegisterStores` inside `BaseApp`, so we don't need to handle store keys inside `DesmosApp`.
See:
https://github.com/cosmos/cosmos-sdk/pull/17284